ftw/tabbedview/browser/resources/tabbedview.js

```diff
@@ -64,6 +64,13 @@ load_tabbedview = function(callback) {
 
     reload_view : function(callback) {
       var params = this.parse_params();
+
+      // add searchbox text property, when input is prefilled
+      // for example, when using the back button
+      if (tabbedview.searchbox.val().length >=3){
```

Comment:
I guess this value is used elsewhere too? right? Probably a variable would make sense.